orchagent/orchdaemon.cpp

```diff
@@ -313,5 +313,6 @@ void OrchDaemon::start()
         for (Orch *o : m_orchList)
             o->doTask();
 
+        flush(); //flush after each event is handled, don't wait
```

Comment:
>//flush after e [](start = 17, length = 15)

Please help follow the coding style of other comments.

Reply:
Sure. Will do